apps/test/unit/lib/kits/maker/sampleSerialPorts.js

```diff
@@ -1,39 +1,28 @@
 /**
  * Sample serial port objects for testing BoardController device detection.
+ * See sample Adafruit board configurations at https://github.com/adafruit/Adafruit_Arduino_Boards/blob/master/boards.txt
  */
 
-export const OSX_DEFAULT_PORTS = [
-  {
-    "comName": "/dev/cu.Bluetooth-Incoming-Port",
-    "serialNumber": "",
-    "pnpId": "",
-    "locationId": "",
-    "vendorId": "0x0",
-    "productId": "0x0"
-  },
-  {
-    "comName": "/dev/cu.Bluetooth-Modem",
-    "serialNumber": "",
-    "pnpId": "",
-    "locationId": "",
-    "vendorId": "0x0",
-    "productId": "0x0"
-  },
+export const CIRCUIT_PLAYGROUND_PORTS = [
+  // Reported by a teacher's Win10 laptop
   {
-    "comName": "/dev/tty.Bluetooth-Incoming-Port",
-    "serialNumber": "",
-    "pnpId": "",
-    "locationId": "",
-    "vendorId": "0x0",
-    "productId": "0x0"
+    comName: "COM5",
+    manufacturer: "Adafruit Circuit Playground",
+    serialNumber: "",
+    pnpId: "",
+    locationId: "",
+    vendorId: "0x239a",
+    productId: "0x8011"
```

Comment:
I was torn about this; on the one hand, this file is sort of a clean, dependencyless fixture and I didn't want to mess with that.  In fact, I wonder if this should somehow be converted to json to show that it's purely data.  On the other hand, DRY...